### PR TITLE
Stream the logs, kinda

### PR DESCRIPTION
**Pull request by proteusvacuum**
Long running jobs give no feedback of what is going on until they are done... 

Peek 2021-02-19 16-12

### CODE REVIEW

File: apps/exports/runner.py [hunk 1]
```diff
@@ -49,23 +56,43 @@ def _run_export_for_project(export_config, project, export_record, force):
...
     export_record.save()
     return export_record
 
 
+def _stream_log(process, export_record):
+    log_buffer = []
+    start_time = datetime.utcnow()
+    while True:
+        output = process.stdout.readline()
+        if process.poll() is not None and output == '':
```
Comment (czue) [inline apps/exports/runner.py:83]: nit: could add a comment saying (i think?) "this means the process is finished and there's nothing more to log". it took me a little while to wrap my head around this code without familiarity of `process` API.

File: apps/exports/runner.py [hunk 2]
```diff
@@ -49,23 +56,43 @@ def _run_export_for_project(export_config, project, export_record, force):
...
+            break
+        if output:
+            log_buffer.append(output.strip())
```
Comment (czue) [inline apps/exports/runner.py:86]: do we ever not want to strip this? e.g. an indented stack trace or something?

**Review by czue — COMMENTED** [review]
This looks great, thanks! I was worried it might be tricky and had been putting it off - glad it wasn't so bad.

The CSS thing, I think we should keep standard (at least for the time being). Other comments are not blockers.

File: static/css/commcare-sync.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+pre.wrapped {
```
[the remark this answers is not in the document]
Comment (proteusvacuum) [reply static/css/commcare-sync.css]: Thanks, I probably should have RTFM... I didn't even see that there was scss files.

File: apps/exports/runner.py (hunk 1 above)
Comment (proteusvacuum) [reply apps/exports/runner.py]: https://github.com/dimagi/commcare-sync/pull/41/commits/5b727328368bd31e2f7ec122532cbdc792dbea8f

File: apps/exports/runner.py (hunk 2 above)
Comment (proteusvacuum) [reply apps/exports/runner.py]: I hadn't thought about that. I think we can just remove the `strip`
https://github.com/dimagi/commcare-sync/pull/41/commits/5b727328368bd31e2f7ec122532cbdc792dbea8f

File: README.md [hunk 4]
```diff
@@ -53,6 +53,12 @@ Then to build (and watch for changes locally) just run:
 npm run dev-watch
 ```
 
+To build the files for production run:
+
+```bash
+npm run build
+```
```
Comment (czue) [inline README.md:60]: :+1: thanks

**Review by czue — APPROVED** [review]
thanks for follow ups

**Pull request merged**